src/ol_infrastructure/infrastructure/applications/dagster/__main__.py

```diff
@@ -110,26 +119,126 @@
     f'dagster-db-access-{env_suffix}',
     name=f'ol-etl-db-access-{env_suffix}',
     description='Access from the data VPC to the Dagster database',
-    ingress=[
-        {
-            'cidr_blocks': [data_vpc_cidr],
-            'ipv6_cidr_blocks': [data_vpc_cidr_v6],
-            'protocol': 'tcp',
-            'from_port': 5432,
-            'to_port': 5432
-        }
-    ],
+    ingress=[ec2.SecurityGroupIngressArgs(
+        cidr_blocks=[data_vpc['cidr'], operations_vpc['cidr']],
+        ipv6_cidr_blocks=[data_vpc['cidr_v6']],
+        protocol='tcp',
+        from_port=5432,
+        to_port=5432
+    )],
     tags=aws_config.tags,
-    vpc_id=data_vpc_id
+    vpc_id=data_vpc['id']
 )
 
 dagster_db_config = OLPostgresDBConfig(
     instance_name=f'ol-etl-db-{env_suffix}',
     password=get_config('dagster:db_password'),
-    subnet_group_name=data_vpc_rds_subnet,
+    subnet_group_name=data_vpc['rds_subnet'],
     security_groups=[dagster_db_security_group],
     tags=aws_config.tags,
     db_name='dagster',
     **defaults(stack)['rds'],
 )
 dagster_db = OLAmazonDB(dagster_db_config)
+
+dagster_db_vault_backend_config = OLVaultPostgresDatabaseConfig(
+    db_name=dagster_db_config.db_name,
+    mount_point=f'{dagster_db_config.engine}-dagster-{dagster_environment}',
+    db_admin_username=dagster_db_config.username,
```

Comment:
Where is the `dagster_db_config.username` defined as I don't see it in dagster_db_config

Reply:
It's coming from the default value defined in the OLDatabaseConfig base class here https://github.com/mitodl/ol-infrastructure/blob/dagster_instance/src/ol_infrastructure/components/aws/database.py#L71